frontend/src/components/CreateProjectDialog.tsx

```diff
@@ -25,16 +24,23 @@ function CreateProjectDialog({}:CreateProjectDialogProps) {
     const handleSubmit = async (e: React.FormEvent) => {
         e.preventDefault();
         setError(ErrorType.None);
-
-        try {
-            // await fetch POST /api/v1/project/:id/?name={projectName}&description={description};
-            closeDialog(); 
-        } catch (err) {
+        const response = await fetch(`/api/v1/project/${projectID}`, {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json',
+            },
+            body: JSON.stringify({
```

Comment:
See https://github.com/ece461l-fa24-final-project/hardwareby.us/blob/6d14b3d36130ba9ba751bdca2396d653c0817c22/backend/src/backend/router.gleam#L116-L128 for impl. The description is a query param. 

We did forget the description field, good catch

Reply:
sorry meant the name field